classes/class4/homework.md

```diff
@@ -0,0 +1,112 @@
+# Homework 4
+In this homework we will start by exploring a new aspect of MongoDB
+and then we will start incorporating AJAX into a web app in order to make it
+more dynamic and easy to use.
+## Reading
+### Mongo Embedding vs Referencing
+Let's say for example that you owned a series of bookstores each with a location,
+a manager, and tons of books. Each book has an author and a price.
+So our data structure so far looks like:
+```
+bookstore
+  location
+  manager
+  book
+    author
+    price
+```
+Keep in mind that a lot of books will be repeated across bookstores.
+How would we convert this to a Mongo datastore?
```

Comment:
Convert what? The question doesn't really hint at what the problem is. I'm reading this like, "we throw it all in an object, duh."

Maybe introduce embedding first (as the "duh" simple solution), since you talk about all the problems that come with it, and then provide referencing as an alternative/solution.